plugins/woocommerce/includes/class-wc-session-handler.php

```diff
@@ -363,9 +363,11 @@ private function get_cache_prefix() {
 	/**
 	 * Save data and delete guest session.
 	 *
-	 * @param int $old_session_key session ID before user logs in.
+	 * @param string|mixed $old_session_key Optional session ID prior to user log-in.  If $old_session_key is not tied
+	 *                                      to a user, the session will be deleted tied to that key will be deleted with the
```

Comment:
I think there's some duplication in this comment, it says "will be deleted" twice.